src/dmd/backend/dwarfdbginf.d

```diff
@@ -228,7 +223,7 @@ static if (ELFOBJ || MACHOBJ)
         static if (ELFOBJ)
             Obj.addrel(seg, offset, I64 ? R_X86_64_32 : R_386_32, cast(int)MAP_SEG2SYMIDX(targseg), val);
         else static if (MACHOBJ)
-            Obj.addrel(seg, offset, null, targseg, RELaddr, cast(uint)val);
+            Obj.addrel(seg, offset, cast(Symbol*) null, targseg, RELaddr, cast(int)val);
```

Comment:
Why the cast ?

Reply:
because addrel is overloaded and the cast guides it to the right overload

Reply:
Then `(Symbol*).init` is IMO better